mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/ConcurrentExistsResponse.test

```diff
@@ -102,7 +102,7 @@ SESSION: 4
 C: 4b CREATE another
 S: \* 1 EXISTS
 S: \* 1 RECENT
-S: \* 1 FETCH \(FLAGS \(\\Flagged \\Recent\)\)
+S: \* 1 FETCH \(FLAGS \(\\Recent\)\)
```

Comment:
Ok don't take seriously this fix, I just want to check if they are other test that fail.
Also if someone could explain me this mpt test, it would be nice because I really do not understand why a just created mailbox would have a message inside it (it's how I interpret this "1 FETCH.............;") and why it should have the \Flaged tag also.